src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -288,6 +288,38 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addColorButton("Line color",
                           QColor(getSettings()->lastMessageColor.getValue()),
                           getSettings()->lastMessageColor);
+    layout.addSeperator();
+    layout.addCheckbox("Gray-out message history before connecting.",
+                       s.grayOutHistory);
```

Comment:
```suggestion
    layout.addCheckbox("Gray-out message history.", s.grayOutHistory);
```
I can't really think of a better name for this, I just cut the `before connecting` part 🤷